src/uct/ib/ud/base/ud_inl.h

```diff
@@ -1,36 +1,78 @@
 #include "ud_log.h"
 
 static UCS_F_ALWAYS_INLINE void
-uct_ud_iface_queue_pending(uct_ud_iface_t *iface, uct_ud_ep_t *ep, int op)
+uct_ud_ep_ctl_op_add(uct_ud_iface_t *iface, uct_ud_ep_t *ep, int op)
 {
     if (ep->tx.pending.ops == UCT_UD_EP_OP_NONE) {
-        ucs_queue_push(&iface->tx.pending_ops, &ep->tx.pending.queue);
+        ucs_arbiter_group_push_elem(&ep->tx.pending.group, 
+                                    &ep->tx.pending.elem);
         ep->tx.pending.ops |= UCT_UD_EP_OP_INPROGRESS;
     }
     ep->tx.pending.ops |= op;
+    ucs_arbiter_group_schedule(&iface->tx.pending_q, &ep->tx.pending.group);
+}
+
+
+static UCS_F_ALWAYS_INLINE void
+uct_ud_ep_ctl_op_del(uct_ud_ep_t *ep, int ops)
```

Comment:
pls add short doc to these inline functions and if possible use more clear names